server/ntb/io/feed_parsers/utils.py

```diff
@@ -34,3 +36,28 @@ def filter_missing_subjects(subjects):
                         'scheme': SUBJECT_CV,
                     })
     return next_subjects
+
+
+def set_default_service(article):
+    if article.get('anpa_category'):
+        return
+    cv = _get_cv(SERVICE_CV)
+    if not cv:
+        return
+    active_items = [item for item in cv.get('items', []) if item.get('is_active')]
+    if active_items:
+        service = active_items[0]
+        for item in active_items:
+            if item.get('qcode') == DEFAULT_SERVICE:
+                service = item
+                break
```

Comment:
You could use `for..else` statement here:

```python
for item in active_items:
    if item.get('qcode') == DEFAULT_SERVICE:
        service = item
        break
else:
    service = active_items[0]
```

that would avoid one assignment.